### ISSUE TITLE
Failure on multiline character literal

### ISSUE THREAD

**Issue by amai2012** [danmar/simplecpp#120]
simplecpp produces a failure
~~~
syntax error: No pair for character ("). Can't process file. File is either invalid or unicode, which is currently not supported.
~~~
on
~~~c
const char*ptr="\\
\n";
~~~
.
gcc/cpp and Visuall Studio accept this.

**Comment by amai2012** [comment]
That issue in turn prevents re-checking https://trac.cppcheck.net/ticket/2435

**Comment by versat** [comment]
Similar issue with multiline text for `#error` message:
https://trac.cppcheck.net/ticket/8444

**Comment by rikardfalkeborn** [comment]
> Similar issue with multiline text for `#error` message:
> https://trac.cppcheck.net/ticket/8444
> 
> Example code:
> 
> ```c
> #ifdef A
> #error "Line \
> Wrap"
> #endif
> ```
> 
> Produces:
> 
> ```
> syntax error: No pair for character ("). Can't process file. File is either invalid or unicode, which is currently not supported.
> ```

This was fixed in 11c6d6f34a9bcc6c62c74a8df73b4488d7ff2711.
